doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,202 @@ You should see something like
      region                <not set>             None    None
 
 Please refer to `this discussion <https://github.com/ray-project/ray/issues/9327>`__ for more details.
+
+
+.. _aws-cluster-cloudwatch:
+
+Using Amazon CloudWatch
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    Disclaimer: CloudWatch integration with Ray requires an AMI (or Docker image) with the Unified CloudWatch Agent pre-installed.
+    AMIs with the Unified CloudWatch Agent pre-installed are provided by the Amazon Ray Team, and are currently available in the us-east-1, us-east-2, us-west-1, and us-west-2 regions.
+    Please direct any questions, comments, or issues to the Amazon Ray Team at `<https://github.com/amzn/amazon-ray/issues/new/choose>`_.
+
+.. list-table:: Unified CloudWatch Agent Images
```

Comment:
Can we also provide an AWS CLI command to run to discover the latest image for a region? This would provide a nice fallback for customers to find the latest AMI prior to the updates being propagated back to this page.